### ISSUE TITLE
React Native Debugg configuration dissapears if using flow

### ISSUE THREAD

**Issue by tafelito** [Microsoft/vscode-react-native#681]
## Actual Behavior
1. if flow-bin is added as a dependency, React Native debug config disappears. It doesn't matter if is installed, just by adding it on package.json (no even need to save) 

![image](https://user-images.githubusercontent.com/5973652/38663521-054dcd9e-3e05-11e8-8b38-0e77b1861599.png)

![image](https://user-images.githubusercontent.com/5973652/38663619-4b0ed896-3e05-11e8-8cc5-0ce047e0fbd7.png)

![image](https://user-images.githubusercontent.com/5973652/38663633-56008d30-3e05-11e8-8f8d-cae0c9086aa5.png)

![image](https://user-images.githubusercontent.com/5973652/38663642-5dbb3f7a-3e05-11e8-9981-eb0023ba4477.png)

## Expected Behavior
1. Show debug config when using flow 

## Software versions
- React-Native VS Code extension version: 0.6.6
- VSCode version: 1.22.1
- OS platform and version: MacOS 10.13.4
- NodeJS version: 
- React Native version: 0.54.4

**Comment by rvntone** [comment]
Same here

**Comment by BarlowTucker** [comment]
Same result if you have 
```
"react-redux": "^5.0.7",
"redux": "^3.7.2",
"redux-thunk": "^2.2.0"
```

**Comment by Yousefjb** [comment]
I have package.json as

```    
{
  "name": "crm",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start"
  },
  "dependencies": {
    "react": "16.3.1",
    "react-native": "0.55.2"
  }
}
```
And debug config does not show

**Comment by annakocheshkova** [comment]
@tafelito @BarlowTucker @Yousefjb thank you for your requests! 
As for me, I didn't manage to reproduce the issue neither with `flow-bin` or `react-redux` and I don't think that having these plugins in `package.json` is the root cause of the issue. 
It seems that the issue you've faced with is a duplicate of #281. You can try the solutions listed there or follow it on the updates.

**Comment by tafelito** [comment]
@annakocheshkova I followed both the issues you mentioned but none of them seems to have a solution. It seems there could be some conflicts with other extensions. I disabled the flow support extension and another one to add react/react native snippets and that make it appeared again. If I enabled back the extensions, rn debug option disappears again. I checked the devtools console but there are no errors there. Anything else I should check?

Not sure if it's related, but as I reported here #529, when I managed to make it appear, when I debug, breakpoints are grayed out, even thou all the extensions.
